services/php-apache/Dockerfile

```diff
@@ -27,7 +27,6 @@ RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - \
     && docker-php-ext-enable opcache \
     && docker-php-ext-enable redis \
     && docker-php-ext-install zip \
-    && docker-php-ext-configure gd --with-freetype --with-jpeg \
```

Comment:
Question: I am manually (in .tugboat/config.yml) install imagemagick libraries (since poets uses imagemagick and not gd). Do we want our images to be more graphics lib neutral, and install both gd and imagemagick in all cases, or do we want to remain standardized on gd, and we customize in our CI scripts for any project that isn't using gd?

Reply:
I don't have a strong opinion. I'd say we leave it at the project level until/if we reach a point where we are configuring it for multiple projects. What do you think?

Reply:
I am honestly split 50/50. 

On the plus side, it costs little in terms of time, and could save a lot of time, to simply build images that have all the libraries that are commonly used in Drupal installations. 

On the minus side, of course, every added library is a potential security hole and will need updates whether used or not. 

I tend to feel imagemagick spreads a lot of cruft into a system (like x11 does) and have always preferred gd to it for that reason. Let's leave as is. Just wanted to flag it since it was something I ended up having to spend a few too many hours trying to figure out.